0015-ilp-addresses/0015-ilp-addresses.md

```diff
@@ -0,0 +1,123 @@
+# ILP Addresses
+
+_ILP addresses_ provide a way route payments to their intended destination through a recursive series of hops, including any number of ILP Connectors. (This happens after the payment is set up on a higher level, such as [SPSP](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md).) Addresses can be subdivided into two categories:
+
+- **Destination Addresses** are complete addresses that can receive payments. A destination address always maps to one account in a ledger. (It can also provide more specific information, such as an invoice ID or a sub-account.) Destination addresses MUST NOT end in a period (`.`) character.
+- **Prefix Addresses** are incomplete addresses representing a group of destination addresses. Many depths of grouping are possible: groups of accounts or sub-accounts, an individual ledger or subledger, or entire neighborhoods of ledgers. Prefix addresses MUST end in a period (`.`) character. Payment setup protocols MUST reject payments to prefix addresses.
+
+Both types of address usually contain one or more period characters as separators.
+
+## Address Requirements
+
+ILP Addresses must meet the following requirements:
+
+1. The address MUST begin with a prefix indicating the allocation scheme. The only supported allocation scheme for now is the [General Allocation Scheme](#general-allocation-scheme), which uses the prefix `g.`.
```

Comment:
Is creating a custom scheme preferable to extending a URI, with restrictions, sort of like [RFC-7565](https://tools.ietf.org/html/rfc7565#section-7) did?

For example, a URI scheme could conform to the restrictions of section 2 below, but would instead look like: `ilp://earth.us.bank.account.bob`